docs/installation.rst

```diff
@@ -52,18 +47,19 @@ Install the latest development version from https://github.com/astropy/pyregion
 
     git clone https://github.com/astropy/pyregion
     cd pyregion
-    python setup.py install
-    python setup.py test
-    python setup.py build_docs
+    pip install -e .[test]
+    pytest
```

Comment:
Good suggestion.  I had tested with --pyargs locally, but good to have in the CI.

Because this repo is not src-layout, I had to use --import-mode=importlib, which pulls the test files themselves from the checkout, but imports the runtime code from site-packages.